bitnami/jaeger/templates/collector/deployment.yaml

```diff
@@ -0,0 +1,203 @@
+            }
+
+            info "Connecting to the Cassandra instance $CQLSH_HOST:$CQLSH_PORT"
+            if ! retry_while "check_cassandra_keyspace_schema" 12 30; then
+              error "Could not connect to the database server"
+              exit 1
+            else
+              info "Connection check success"
+            fi
+        env:
+        - name: CQLSH
+          value: /opt/bitnami/cassandra/bin/cqlsh
+        - name: CQLSH_HOST
+          value: {{ printf "%s-cassandra" (include "common.names.fullname" .) }}
+        - name: CQLSH_PORT
+          value: {{ .Values.cassandra.containerPorts.cql | quote }}
+        - name: CASSANDRA_USERNAME
+          value: {{ .Values.cassandra.dbUser.user }}
+        - name: CASSANDRA_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: {{ printf "%s-cassandra" (include "common.names.fullname" .) }}
+              key: cassandra-password
+        - name: CASSANDRA_KEYSPACE
+          value: {{ .Values.cassandra.keyspace }}
+      {{- if .Values.collector.initContainers }}
+      {{- include "common.tplvalues.render" (dict "value" .Values.collector.initContainers "context" $) | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: jaeger-collector
+          image: {{ include "jaeger.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          {{- if .Values.collector.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.collector.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.command "context" $) | nindent 12 }}
+          {{- else if .Values.collector.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.collector.command "context" $) | nindent 12 }}
+          {{- else }}
+          command: ["/opt/bitnami/jaeger/bin/jaeger-collector"]
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.args "context" $) | nindent 12 }}
+          {{- else if .Values.collector.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.collector.args "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.collector.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.collector.lifecycleHooks "context" $) | nindent 12 }}
+          {{- end }}
+          ports:
+            - name: zipkin
+              containerPort: {{ .Values.collector.service.ports.zipkin }}
+            - name: binary
+              containerPort: {{ .Values.collector.service.ports.binary }}
+            - name: grpc
```

Comment:
How are these ports changed when running the jaeger-collector binary? I would imagine that in the args section you could set these ports. Otherwise it would make no sense allowing editing them

Reply:
Need to map then though env vars. Completely missed that. WIP.

Reply:
Will follow ports discussion on https://github.com/bitnami/charts/pull/13480#discussion_r1032314451

Reply:
I still do not see how you configure the ports for zipkin, binary and grpc. I don't see any  reference to the env vars that set these ports